### ISSUE TITLE
window.navigator.vibrate() missing

### ISSUE THREAD

**Issue by mnordine** [dart-lang/sdk#43302]

**Comment by keertip** [comment]
@mnordine , is this to do with `dart:html`? Flutter? If Flutter related, please open an issue with details in the [flutter issue tracker](https://github.com/flutter/flutter/issues).

Closing, not enough details to work on. Reopen with details if needed.

**Comment by mnordine** [comment]
@keertip not sure why you'd need more info? All the info is there in the api call. It's literally the same as the javascript call.

**Comment by sigmundch** [comment]
@mnordine - thanks for the report!

Note that @keertip request was to help the triage process. We track a lot of areas in this repo (vm, analyzer, dart2js, ddc, libraries), and sometimes we get reports that are flutter-specific. We use a rotation to triage and assign to subareas and sometimes we find it helpful to ask for more details to make it easier to route the issue to the right subteam.

**Comment by mnordine** [comment]
I can't add labels to help triage, can I?

**Comment by sigmundch** [comment]
You are correct, I don't believe that's currently available outside the organization :-( (@kevmoo might know better here, but he is on vacation currently)
